### PR TITLE
Backend Djoser Settings

### PR DESCRIPTION
**Pull request by melihaydogd**
Djoser settings are done. User can be created and an activation mail will be sent to the user.

### CODE REVIEW

**Comment by mertlkn** [comment]
Can we add to settings.py:

SIMPLE_JWT = {
    'AUTH_HEADER_TYPES': ('JWT',),
    'BLACKLIST_AFTER_ROTATION': False,
}

because otherwise i'm getting an error on the jwt side.

**Review by mertlkn — APPROVED** [review]

**Review by zcanfes — CHANGES_REQUESTED** [review]
Other than what I commented below, everything seems fine. The email templates look very good and the code works correctly.

File: postory/backend/user_endpoint/models.py [hunk 1]
```diff
@@ -4,23 +4,23 @@
 from django.contrib.auth.models import AbstractBaseUser,PermissionManager,BaseUserManager, PermissionsMixin
 
 class UserAccountManager(BaseUserManager):
-    def create_user(self,username,name,surname,email,password=None):
-        if not username:
+    def create_user(self, email, password=None, **extra_fields):
+        if not email:
             raise ValueError('Users must have an username')
```
[the remark this answers is not in the document]
Comment (melihaydogd) [reply postory/backend/user_endpoint/models.py:9]: Yes, you are right. Actually, there is no need for this check. The fields will be checked on frontend.

**Review by zcanfes — APPROVED** [review]
Everything looks to be working fine :)

**Pull request merged**